### DISCUSSION TITLE
Any plans for a new release?

### DISCUSSION THREAD

**Discussion by spoofedpacket** [go-graphite/carbonapi#792] [General]
Hey folks, do you have plans for a new release any time soon? 

In particular we're looking to make use of the `expand()` function but it's only in mainline currently
https://github.com/go-graphite/carbonapi/pull/748

Thanks!

**Comment by reyjrar** [comment]
Came here to attain the status of master, ie, is it safe to deploy to production?

**Reply by Civil** [reply]
So about the status of master - I'm not running carbonapi on any production-scale workloads, just a few test environments. So I rely on feedback from people to check on status of main branch and on how ready it is for a new release (mostly).

So far it seems it is not worse than previous stable release, so I'll cut a new point release once I'll update Go version on build images. I don't have any specific ETA for that, but I'll try to do that during upcoming weekend (26-27 Aug)

**Comment by Civil** [comment]
Not sure how notification in GitHub discussions works, so I'll copy part of the reply from thread above:
I'll cut a new point release once I'll update Go version on build images. I don't have any specific ETA for that, but I'll try to do that during upcoming weekend (26-27 Aug). And I don't expect it to change from main branch, except for Changelog (which I also need to update before cutting the release)
